src/OrchardCore.Modules/OrchardCore.Templates/Controllers/TemplateController.cs

```diff
@@ -76,16 +76,14 @@ public async Task<IActionResult> Index(PagerParameters pagerParameters, string d
 
             var count = templatesDocument.Templates.Count;
 
-            IEnumerable<KeyValuePair<string, Template>> templates;
+            IEnumerable<KeyValuePair<string, Template>> templates = templatesDocument.Templates;
             if (!string.IsNullOrEmpty(displayText))
-                 templates = templatesDocument.Templates.Where(template => template.Key.Contains(displayText, StringComparison.InvariantCultureIgnoreCase));
-            else
-                templates = templatesDocument.Templates;
-
-             templates
-                 .OrderBy(x => x.Key)
-                 .Skip(pager.GetStartIndex())
-                 .Take(pager.PageSize);
+               templates = templates.Where(template =>
+                    template.Key.Contains(displayText, StringComparison.InvariantCultureIgnoreCase));
```

Comment:
Továbbra is legyen Orchardban `if`-ek körül zárójel.

Reply:
ahha, hiányzik a { } jelek.